src/api/docker-compose.yml

```diff
@@ -106,3 +106,52 @@ services:
       - 'traefik.http.routers.login.rule=Host(`login.localhost`)'
       # Specify the login port
       - 'traefik.http.services.login.loadbalancer.server.port=8080'
+      # Add middleware to this route to strip the /v1/posts prefix
+      - 'traefik.http.middlewares.strip_api_prefix.stripprefix.prefixes=/${API_VERSION}/image'
+      - 'traefik.http.middlewares.strip_api_prefix.stripprefix.forceSlash=true'
+      - 'traefik.http.routers.image.middlewares=strip_api_prefix'
+
+  redis:
+    image: redis:latest
+    container_name: api_redis
+    ports:
+      - ${REDIS_PORT}
+    command: ['redis-server', '--appendonly', 'yes']
+    volumes:
+      - ../../redis-data:/data
```

Comment:
I wonder if we should move this volume into `src/api`?

Reply:
I can move it there and remove it from the root folder.

Reply:
Don't remove that one, it's our legacy one that's being used by the current setup.  I'm just thinking about isolating these two from each other while we get the services working.